### PR TITLE
autofocus task navigation

### LINKED ISSUE #2377: autofocus for text tasks
When we have a text task, can we have it "autofocus" on the text box - would save people from clicking. (I've created a project with just one task - entering text. Having to click each time really kills the rhythm.)

### PR DESCRIPTION
**Pull request by simensta**
This PR improves the keyboard navigation of the following tasks in the `/classify` page:
- text tool
- single answer
- drawing
- multiple answer

This should apply to regular tasks, as well as subTasks. Here is the example branch with a [simple workflow ](https://autofocus-task-navigation.pfe-preview.zooniverse.org/projects/markb-panoptes/focus-testing-project/classify?workflow=2304)and a [combo-task workflow](https://autofocus-task-navigation.pfe-preview.zooniverse.org/projects/markb-panoptes/focus-testing-project/classify?workflow=2305). Fixes #2377. 

~~WIP~~
- [x] issues in Safari (solution: turn on keyboard navigation in system preferences)

@eatyourgreens, @mcbouslog -- thoughts?

### CODE REVIEW

File: app/classifier/drawing-tools/root.cjsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ module.exports = React.createClass
           {for detailTask, i in toolProps.details
             detailTask._key ?= Math.random()
             TaskComponent = tasks[detailTask.type]
-            <TaskComponent key={detailTask._key} task={detailTask} annotation={toolProps.mark.details[i]} onChange={@handleDetailsChange.bind this, i} />}
+            <TaskComponent autoFocus={true if i is 0} key={detailTask._key} task={detailTask} annotation={toolProps.mark.details[i]} onChange={@handleDetailsChange.bind this, i} />}
```
Comment (eatyourgreens) [inline app/classifier/drawing-tools/root.cjsx]: This can probably just be `autoFocus={i is 0}`

**Comment by eatyourgreens** [comment]
Looks good to me. One small syntax comment about whether `autoFocus` should be `true/false` or `true/undefined`.

File: app/classifier/drawing-tools/root.cjsx (hunk 1 above)
Comment (simensta) [inline app/classifier/drawing-tools/root.cjsx]: good catch -- thanks!

**Comment by eatyourgreens** [comment]
@mcbouslog @simensta Looks like there might still be some issues with autofocus and `select` elements in Safari, but I'm happy to get this merged and deal with those separately.

**Pull request merged**